src/PIXMAN/Pixman.hsc

```diff
@@ -0,0 +1,35 @@
+{-# LANGUAGE EmptyDataDeriving, PatternSynonyms #-}
+
+module PIXMAN.Pixman where
+
+#include <pixman.h>
+
+import Foreign
+import Foreign.C.Types
+
+type PIXMAN_region32_t = PIXMAN_region32
+type PIXMAN_region32_data_t = PIXMAN_region32_data
+type PIXMAN_box32_t = PIXMAN_box32
```

Comment:
What is the point of these type synonyms?

Reply:
`pixman.h` uses these typedefs. I figured that I may as well mirror that for clarity.

Reply:
Some C programmers like to make those sorts of typedefs in order to avoid repeating the keyword `struct` all the time: `pixman_region32_t` is defined as a synonym for `struct pixman_region32`, and so on. But I feel that’s unnecessary in Haskell.

Reply:
I'm going to leave this as-is. We need to bind to the structs as they are defined, without the `_t`, but the `_t` version is used all over the place so I think that there will be less confusion with the alias.